lib/net/networking_engine.js

```diff
@@ -395,7 +400,7 @@ shaka.net.NetworkingEngine = class extends shaka.util.FakeEventTarget {
 
       // Override the original URI to make the scheme explicit.
       uri.setScheme(scheme);
-      request.uris[index] = uri.toString();
+      uriToRequest = uri.toString();
```

Comment:
This is a change in behavior.  Before, we would update the uris list in the request.  With this, we would not any more.